source/polling-srv-records-for-mongos-discovery/tests/README.rst

```diff
@@ -126,3 +126,14 @@ mock the addition of the following DNS record::
 
 Wait until ``2*rescanSRVIntervalMS`` and assert that the final topology description
 only contains one server: ``localhost.test.build.10gen.cc.`` at port ``27017``.
+
+10. Test that SRV polling is done correctly for rawsrv URIs
+-----------------------------------------------------------
+
+Connect to ``mongodb+rawsrv://_customname._tcp.test3.test.build.10gen.cc/``,
```

Comment:
I'll have to speak with the BUILD team about setting up a custom SRV record with `customname` as a service name for this prose test to work. Will update with a ticket number on this thread.